README.md

```diff
@@ -7,6 +7,20 @@ Tools for managing, configuring and monitoring SONiC
 # CII Best Practices
 [![CII Best Practices](https://bestpractices.coreinfrastructure.org/projects/3933/badge)](https://bestpractices.coreinfrastructure.org/projects/3933)
 
+# Docker Container Setup
+Use the `setup-container.sh` script to automatically create and configure your sonic-mgmt Docker container. This script should be run directly from the root of the sonic-mgmt repo. You should also run this script as the user that will be using the created container.
```

Comment:
how is this setup instruction related to the vssetup readme.md in the doc directory?

Reply:
I don't believe they are really that related. The VSSetup document only applies to the virtual switch testbed, but the container created using this script is able to be used with physical devices. @daall please confirm/deny, I'm not entirely sure

Reply:
Nevermind, after talking with @daall I will update the VS Setup Readme to use this script so that the container setup is consistent.

Reply:
I am little confused with the motivation of this PR. Can someone help me clear this confusion?

Do these changes add support for:
1. using docker VS instead of VM-based VS in the Vswitch test bed (or)
2. keeping the vm-based VS in the VS testbed but using docker VS instead of cEOS dockers for the other nodes

Apologies if this is covered elsewhere. I was unable to find it and understand the context. Thanks.

Reply:
Sorry for any confusion. This docker container is not meant to replace the VS testbed or any other testbed. The container created by this script is for the `docker-sonic-mgmt.gz` image, and is used for running tests on testbeds (both physical and virtual).